shared/actions/fs/platform-specific.desktop.js

```diff
@@ -169,13 +169,16 @@ const fuseStatusToActions = (previousStatusType: 'enabled' | 'disabled' | 'unkno
     ? [
         FsGen.createSetDriverStatus({
           driverStatus: Constants.makeDriverStatusEnabled({
-            dokanOutdated: status.installAction === 2,
+            dokanOutdated: status.installAction === 2, // 2 == UPGRADE
```

Comment:
I think you meant to use this:
https://github.com/keybase/client/blob/970bb5d56e9ad815846eb3b2a0e246b84232f6d8/shared/constants/types/rpc-gen.js#L490

`status.installAction === RPCTypes.installInstallAction.upgrade`